inference-engine/src/transformations/include/transformations/common_optimizations/make_stateful.hpp

```diff
@@ -0,0 +1,5 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/pass/make_stateful.hpp>
```

Comment:
This is new transformation so we don't need this header.

Reply:
done